NeuroID/src/androidLib/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -55,14 +55,27 @@ class NeuroID private constructor(
     }
 
     fun setUserID(userId: String) {
+        val gyroData = NIDSensorHelper.getGyroscopeInfo()
+        val accelData = NIDSensorHelper.getAccelerometerInfo()
+
         application?.let {
             NIDSharedPrefsDefaults(it).setUserId(userId)
         }
         getDataStoreInstance().saveEvent(
             NIDEventModel(
                 type = SET_USER_ID,
                 uid = userId,
-                ts = System.currentTimeMillis()
+                ts = System.currentTimeMillis(),
+                gyro = NIDSensorModel(
```

Comment:
We can return NIDSensorModel on NIDSensorHelper.get Gyroscope Info() and avoid all that instances created